tests/integration/test_api.py

```diff
@@ -78,6 +79,17 @@ def test_download(tmp_path, selection, use_url):
     assert all(Path(f).exists() for f in files)
 
 
+@pytest.mark.parametrize("system", [PROD, UAT])
+def test_earthdata_status(system):
+    result = earthaccess.status(system)
```

Comment:
We don't want to make real network calls in unit tests. For HTTP testing, make use of the `responses` library. For guidance, see `tests/unit/test_auth.py`, but if you need further assistance, let me know.

We want to test the following responses, at a minimum:

- happy path: we get a "normal" response (with status code 200)
- status 200, but malformed json in the response (should return the "defaults" mapping defined in the `status` function, due to json decoding error)
- non-200 status, say 500, (should also return "defaults", due to raising for status)

Reply:
I would use a whole real response if it were me!

Reply:
@Sherwin-14, parametrize the json value along with the system value, using a full json response. This will allow you to provide a PROD response value that is different from a UAT response value.